fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -414,11 +439,14 @@ private LucenePartitionInfoProto.LucenePartitionInfo newPartitionMetadata(long t
      * get most recent partition's info.
      *
      * @param groupKey group key
+     * @param context the context in which to execute; should generally be {@code state.context}
+     * @param indexSubspace the index subspace; should generally be {@code state.indexSubspace}
      * @return partition metadata future
      */
     @Nonnull
-    private CompletableFuture<LucenePartitionInfoProto.LucenePartitionInfo> getNewestPartition(@Nonnull Tuple groupKey) {
-        return getEdgePartition(groupKey, true);
+    private static CompletableFuture<LucenePartitionInfoProto.LucenePartitionInfo> getNewestPartition(
```

Comment:
Made this static because some code needs to access this from, potentially, other transactions.